source/Outfit.cpp

```diff
@@ -118,11 +118,25 @@ const vector<string> Outfit::CATEGORIES = {
 
 void Outfit::Load(const DataNode &node)
 {
-	if(node.Size() >= 2)
+	if(node.Size() == 2)
 	{
 		name = node.Token(1);
 		pluralName = name + 's';
 	}
+	else if(node.Size() >= 3)
+	{
+		name = node.Token(2);
+		pluralName = name + 's';
+		
+		const Outfit *parent = GameData::Outfits().Get(node.Token(1));
+		
+		// Copy data over from the parent outfit.
+		category = parent->Category();
+		thumbnail = parent->Thumbnail();
+		cost = parent->Cost();
+		mass = parent->Mass();
+		attributes = parent->Attributes();
+	}
```

Comment:
We can't guarantee that this information is available, as the parent might not be loaded by this point. (That is, the variant might be defined before the parent.) If this is to be done, then we need a FinishLoading function that is only called after everything is loaded, akin to what Ship.h/cpp has.